iceoryx_posh/test/moduletests/test_popo_trigger.cpp

```diff
@@ -492,108 +606,133 @@ TEST_F(Trigger_test, ResetEventBasedTriggerIsNotValidAndNotTriggered)
     sut.reset();
 
     EXPECT_FALSE(sut.isValid());
-    EXPECT_FALSE(sut.hasTriggered());
+    EXPECT_FALSE(sut.isStateConditionSatisfied());
     EXPECT_THAT(sut.getUniqueId(), Eq(Trigger::INVALID_TRIGGER_ID));
     EXPECT_THAT(sut.getTriggerType(), Eq(TriggerType::INVALID));
 }
 
 TEST_F(Trigger_test, ValidEventBasedTriggerIsLogicalEqualToSameEventOriginAndEmptyHasTriggeredCallback)
```

Comment:
I think the names of this test and of some of the following are wrong now. The HasTriggeredCallback is not empty anymore after your refactoring of the tests and maybe I overlooked it but I think there is no test for an empty hasTriggeredCallback anymore.